include/boost/mpi/detail/forward_iprimitive.hpp

```diff
@@ -42,7 +42,7 @@ class forward_iprimitive
     
     /// loading of arrays is forwarded to the implementation archive
     template<class T>
-    void load_array(serialization::array<T> & x, unsigned int file_version )
+    void load_array(serialization::array_wrappe<T> & x, unsigned int file_version )
```

Comment:
On 25/04/2016 09:17, Vladimir Prus wrote:
>
> In include/boost/mpi/detail/forward_iprimitive.hpp 
> <https://github.com/boostorg/mpi/pull/30#discussion_r60870196>:
>
> > @@ -42,7 +42,7 @@ class forward_iprimitive
> >
> >      /// loading of arrays is forwarded to the implementation archive
> >      template<class T>
> > -    void load_array(serialization::array<T> & x, unsigned int file_version )
> > +    void load_array(serialization::array_wrappe<T> & x, unsigned int file_version )
>
> Missing 'r'? Is this code exercised by any tests?
>

Does not ring any bell. Wasn't even aware serialization arrays were 
renamed or handled.

Alain